### DISCUSSION TITLE
Installing Elementary 8 blank screen

### DISCUSSION THREAD

**Discussion by ephodges** [elementary/triage#511] [Q&A]
I tried multiple times installing Elementary 8 from just about all the releases and it gets as far as showing a blurred Elementary logo and the a black screen with a curser. Please help

**Comment by marqu3s** [comment]
I had this problem also, but they solved it a week ago (or more). The daily build is booting and installing on the latest VMware Player. But I'm not getting a dock visible.

![image](https://github.com/elementary/triage/assets/2284983/c8db61c1-ed7a-48c4-bb0b-52e9a8d8f724)

And if I try Pantheon (wayland) it doesn't show the desktop, just a black screen with the cursor blinking on the top left.

**Reply by ephodges** [reply]
I got it working with a newer release. I'm having the same issue with the dock. I tried searching for it but no luck.

**Reply by ryonakano** [reply]
The absence of the dock is tracked at https://github.com/elementary/dock/issues/213
